src/libsystemd/sd-journal/mmap-cache.c

```diff
@@ -64,11 +64,13 @@ struct MMapCache {
 
         LIST_HEAD(Window, unused);
         Window *last_unused;
+        unsigned n_unused;
 
         Window *windows_by_category[_MMAP_CACHE_CATEGORY_MAX];
 };
 
 #define WINDOWS_MIN 64
+#define UNUSED_MIN 8
```

Comment:
Picking the right value here is up for grabs.

@mischief if you're up for doing a similar set of timings like you did for the FD_WINDOWS_MIN in the closed PR, it could help inform what's the best bang for buck here... TIA